templates/help.html

```diff
@@ -97,7 +96,7 @@ <h2 id="account-management">Account Management</h2>
   </p>
   <h2 id="downloading-files">Downloading Files</h2>
   <p>
-    You can download files by clicking on the link in the file entry. You can also bulk download files by selecting multiple files using the checkbox and then clicking the Download button. This will give you a ZIP file with all the subtitles you selected.
+    You can download files by clicking on the link in the file entry. You can also bulk download files by selecting multiple files using the checkbox and then clicking the Download button. This will give you a ZIP file with all the subtitles you selected. Shift clicking the checkbox to select multiple files is also supported.
```

Comment:
```suggestion
    You can download files by clicking on the link in the file entry. You can also bulk download files by selecting multiple files using the checkbox and then clicking the Download button. This will give you a ZIP file with all the subtitles you selected. You can also Shift click the checkbox to select multiple files in a range.
```